packages/api-core/src/me-preferences/types.ts

```diff
@@ -30,6 +30,7 @@ export interface UserPreferences {
 	'hosting-dashboard-opt-in'?: HostingDashboardOptIn;
 	[ key: `hosting-dashboard-overview-storage-notice-dismissed-${ number }` ]: string | undefined; // Timestamp when the user dismissed the notice
 	[ key: `hosting-dashboard-tours-${ string }` ]: string; // ISO date string when the user completed the tours
+	[ key: `time-mismatch-warning-dismissed-${ number }` ]: string | undefined; // Timestamp when the user dismissed the notice
```

Comment:
Let's add `dashboard-` prefix, e.g.


```suggestion
	[ key: `dashboard-site-time-mismatch-warning-dismissed-${ number }` ]: string | undefined; // Timestamp when the user dismissed the notice
```

Reply:
In that case I wonder if it should then be `hosting-dashboard-` rather than `dashboard`, as with the surrounding examples?

Reply:
Sorry, yeah makes more sense.

Reply:
Thanks! Added in 548689fdabc9ea058acc7254ac5ac0fb85cce982